user_guides/connector/src/main/jbake/content/attributes.conf

```diff
@@ -1,10 +1,11 @@
 :TechnologyFullName: Jakarta Connectors
 :TechnologyShortName: Connectors
 :LegacyAcronym: JCA		   
-:TechnologyVersion: 1.7
-:TechnologyRI: Eclipse GlassFish 5.1
+:TechnologyVersion: 2.0
+:TechnologyRI: Eclipse GlassFish 6.0
+:PlatformVersion: 9
```

Comment:
https://github.com/eclipse-ee4j/jakartaee-tck/pull/447 was merged using "JakartaEEVersion" as the platformversion for all TCKs. For the sake of uniformity can we keep "JakartaEEVersion" instead of "PlatformVersion" which is already present in attributes.conf

Reply:
Done :rocket: